source/simulator/solver_schemes.cc

```diff
@@ -709,7 +710,9 @@ namespace aspect
                                                             dcr.residual,
                                                             dcr.residual_old);
 
-                pcout << "   The linear solver tolerance is set to " << parameters.linear_stokes_solver_tolerance << std::endl;
+                pcout << "   The linear solver tolerance is set to "
+                      << parameters.linear_stokes_solver_tolerance
+                      << ". ";
```

Comment:
You removed an `std::endl` at the end of this line. Was this on purpose? If it was: Can you post how the output looked like before and now? I just want to check if it makes sense to remove the newline in this context.